packages/trpc/server/routers/viewer.tsx

```diff
@@ -158,7 +158,7 @@ const publicViewerRouter = router({
       }
     }),
   // REVIEW: This router is part of both the public and private viewer router?
-  slots: slotsRouter,
+  // slots: slotsRouter,
   cityTimezones: publicProcedure.query(async () => {
     /**
      * Lazy loads third party dependency to avoid loading 1.5Mb for ALL tRPC procedures.
```

Comment:
Would it make sense to split this specific endpoint to another router? Since it seems like dynamic importing doesn't actually reduce bundle size [according to these findings](https://calcominc.slack.com/archives/C04H3D79LAK/p1681235764477849?thread_ts=1681232021.113919&cid=C04H3D79LAK).

Reply:
<img width="787" alt="image" src="https://user-images.githubusercontent.com/3504472/231325845-c324a1bc-b2e8-4198-9b72-1663cd704da2.png">